json-logs/samples/api/conversations.inviteShared.json

```diff
@@ -0,0 +1,10 @@
+{
+  "ok": false,
+  "error": "",
+  "url": "https://www.example.com/",
+  "invite_id": "I00000000",
+  "conf_code": "",
+  "is_legacy_shared_channel": false,
+  "needed": "",
```

Comment:
When are the `needed` and `provided` fields returned by the API or they a feature of the java-sdk impl? I tried playing around with the api methods tester to generate these fields, but I didn't see these returned.

Reply:
@srajiang They can be returned when your app does not sufficient scopes for an API call. The fields are defined for all the response types in this SDK: 

* https://github.com/slackapi/java-slack-sdk/blob/v1.10.0/slack-api-client/src/main/java/com/slack/api/methods/SlackApiTextResponse.java
* https://github.com/slackapi/java-slack-sdk/blob/v1.10.0/slack-api-client/src/main/java/com/slack/api/methods/response/auth/AuthTestResponse.java#L9-L13